src/dtos/types.ts

```diff
@@ -209,5 +209,6 @@ export interface IMetadata {
 export type ISplitFiltersValidation = {
   queryString: string | null,
   groupedFilters: Record<SplitIO.SplitFilterType, string[]>,
-  validFilters: SplitIO.SplitFilter[]
+  validFilters: SplitIO.SplitFilter[],
+  originalFilters: SplitIO.SplitFilter[]
```

Comment:
@emmaz90 , as we discussed earlier, the `originalFilters` object seems that is not necessary. We can use the `validFilters` array to count the total number of flag sets. 
Please, roll back the changes. Thank you :)

Reply:
Rolled back and fixed validFilters beaviour 👏  thank you!